pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools", "wheel"]  # PEP 508 specifications.
+
+
+[tool.black]
+line-length = 120
+target_version = ['py37', 'py27', 'py36']
```

Comment:
remove py27?

Reply:
I kept it because we still have py27 support :(. So wasn't comfortable making formatting related this and risk breaking it. I just quickly ran it and the only changes were removing `u` from strings like `u"help"`. For now, I would prefer to just leave it.